Source/ALS/Public/AlsAnimationInstance.h

```diff
@@ -44,7 +44,7 @@ class ALS_API UAlsAnimationInstance : public UAnimInstance
 
 	// Time of the last teleportation event.
 	UPROPERTY(VisibleAnywhere, BlueprintReadOnly, Category = "State", Transient, Meta = (ClampMin = 0))
-	float TeleportedTime;
+	double TeleportedTime;
```

Comment:
Did you said that you want it to be `double`?

Reply:
Yes, that's right